tests/test-common.js

```diff
@@ -1006,6 +1008,48 @@ describe('JSON-LD Signatures', () => {
           };
           assert.deepEqual(result, expected);
         });
+        it('should sign and verify with out a controller passed to purpose',
```

Comment:
```suggestion
        it('should sign and verify without a controller passed to purpose',
```

Reply:
There are already tests that sign and verify without a controller passed to purpose. Should this say: 'should sign and verify using a key that has a controller object'? Or something to that effect?